pkg/tsdb/tempo/trace.go

```diff
@@ -128,3 +133,39 @@ func (s *Service) createRequest(ctx context.Context, dsInfo *Datasource, traceID
 	req.Header.Set("Accept", "application/protobuf")
 	return req, nil
 }
+
+func decode(encoding string, original io.ReadCloser) ([]byte, error) {
```

Comment:
Is it a copy-paste from [xxx](https://github.com/grafana/grafana/blob/e18d029a241b2515fdeab8e6c61a8608d474663b/pkg/tsdb/cloud-monitoring/resource_handler.go#L191)? I think that might have been added to support "brotli". I thought https://pkg.go.dev/net/http#Transport would handle decompression og gzip and deflate out of the box 🤔